### ISSUE TITLE
operator: driver not functional in direct mode

### ISSUE THREAD

**Issue by pohly** [intel/pmem-csi#581]
When using the operator to deploy PMEM-CSI in direct mode, the node driver fails:
```
pmem-csi-node-cwn2w/pmem-driver: failed to run driver: FATAL: /sys mounted read-only, can not operate
```

**Comment by pohly** [comment]
A test for this is in https://github.com/intel/pmem-csi/pull/575, look for `operator-direct-production driver runs`.

**Comment by avalluri** [comment]
I pushed the fix in #584

**Comment by pohly** [comment]
It should work now, but we still need tests for it.
